tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -40,20 +40,6 @@ def teardown():
     return es
 
 
-@pytest.fixture(scope="function")
-def ripsaw(request):
-
-    # Create RipSaw Operator
-    ripsaw = RipSaw()
-
-    def teardown():
```

Comment:
teardown should be revisited to ensure it runs

Reply:
set it back to be fixture